src/modules/esl-event-listener/core/api.ts

```diff
@@ -41,9 +40,7 @@ export class ESLEventUtils {
    * @param host - host object (listeners context) to associate subscription
```

Comment:
Mb replace this comments to ESLEventListener and for unsubsrive remove as doubled.

Reply:
Thanks!
Need to check if they automatically inherited in IDE spelling in case yes, we can consider shortcutting descriptions here
Add to the todo list for the follow ups